internal/gke/nanny.go

```diff
@@ -246,7 +246,7 @@ func RunDistributor(ctx context.Context, port int) error {
 		return err
 	}
 
-	// Track http clients to the babysitters, keyed by replica set.
+	// Track http clients to the babysitters, keyed by replica set and the babysitter address.
 	var mu sync.Mutex
 	bsHttpClients := map[string]endpoints.Babysitter{}
```

Comment:
Can you create an inlined struct:

```
type clientKey struct {
  replicaSet, babysitterAddr string
}
```

, and then key the map by this:

```
bsHttpClients := map[clientKey]endpoints.Babysitter{}
```

This would be more explicit and type-safe too.

Reply:
Much better, thank you